src/VisualStudio/Core/Def/DocumentOutline/DocumentOutlineView.xaml

```diff
@@ -77,22 +77,16 @@
         </Grid.RowDefinitions>
         <ContentControl Grid.Row="0" Name="Commands" Focusable="True" />
         <Grid Grid.Row="1" x:Name="SearchHost" />
-        <!-- Two important properties are being set for TreeView:
-               We set IsVirtualizing to "True" so that WPF only generates internal data-structures elements that are visible.
-               Setting VirtualizationMode to "Recycling" ensures that WPF internal data is reused as items scroll in and out of view.
-        -->
```

Comment:
Should this comment get moved to where the properties are still set?

Reply:
It seemed implicit in the new name, but I can include them.

Reply:
➡️ Done